openstates/cli/update.py

```diff
@@ -399,7 +399,11 @@ def _tb_info(type, value, tb):  # type: ignore
     logging.info(f"Module: {args.module}")
 
     # delete all objects from S3 folder for current module
-    delete_all_objects_from_s3_folder(args.module)
+    try:
+        delete_all_objects_from_s3_folder(args.module)
+    except Exception as e:
```

Comment:
Is it worth putting the exception within the `delete_all_objects_from_s3_folder` function instead? Doing that would let us indicate _which_ objects were deleted (or which failed to delete).
Alternatively, returning a list of deleted objects might be useful? Whether we drop that information or not, we may want it later?

Reply:
Makes sense, I'd add this now.